scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorThread.java

```diff
@@ -97,21 +97,23 @@ private void doWork() {
 
   private void executeCurrentTask() {
     final var properties = currentTask.getActor().getContext();
-    MDC.setContextMap(properties);
+    boolean resubmit = false;
+
     MDC.put("actor-scheduler", actorThreadGroup.getSchedulerName());
+    for (final var property : properties.entrySet()) {
+      MDC.put(property.getKey(), property.getValue());
+    }
 
     idleStrategy.onTaskExecuted();
 
-    boolean resubmit = false;
-
-    final var actorName = properties.getOrDefault("actor-name", "");
-    try (final var ignored = MDC.putCloseable("actor-name", actorName)) {
```

Comment:
❓ This is confusing why we did this before like this ? 🤔

Reply:
It's what they recommend you do, but it generates garbage =/